src/opentime/rationalTime.h

```diff
@@ -263,11 +291,6 @@ class RationalTime
     static RationalTime     _invalid_time;
     static constexpr double _invalid_rate = -1;
 
-    RationalTime _floor() const noexcept
-    {
-        return RationalTime{ floor(_value), _rate };
-    }
-
```

Comment:
Please excuse my lack of C++ foo. 
Is a method prefixed with an "_" meant to be "private" or not encouraged for use outside the class in C++?
I'm asking as replacing this could potentially cause some issues for people using it?

Reply:
The underbar means "private" although it's not a general C++ convention. It's common in film, on linux/mac. @darbyjohnston I didn't think about the name when I approved, just the removal of the underbar. Is there a reason to add _value() to all the functions? floor() ceil() and round() are class methods so they won't conflict with the standard library.

Reply:
Ah, I see the conversation. For the record, @jminor suggested them, and I said that sounded fine to me :) Josh's reasoning was that floor is taking the floor of the value. But now that time has passed, and I read it without context, value is such a generic term, that it is no longer intuitive to me that floor_value() means take the floor of the value member. I'll defer to Josh on this one ~ I'm sure I will be confused a couple more times until it sticks in my brain.

Reply:
I think that's a point in favor of the shorter names, when you looked at it fresh that's what you expected to see. I personally think the "_value" is implicit and if we need additional rounding functions they can have more explicit names:
```
RationalTime floor() const;
RationalTime floor_rate() const;
RationalTime floor_value_and_rate() const;
```

Reply:
I reverted back to the shorter names, hopefully it reads better and I think operating on the value is what people would expect.